packages/cojson/src/localNode.ts

```diff
@@ -240,6 +311,10 @@ export class LocalNode {
                 );
             }
 
+            for (const peer of serverPeers) {
+                void node.syncManager.requestSyncStateHydration(peer.id);
+            }
```

Comment:
We should define two new possible states for the `localNode`: "bootstrapping" and "ready"

The sync state hydration should start when we are ready so we won't need a timeout inside of it.